src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,50 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class SetTest {
+
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @DisplayName("요구사항1 - Set의 size() 메소드를 활용해 Set의 크기를 확인하는 학습테스트")
+    @Test
+    void size() {
+        // given & when & then
+        assertThat(numbers).hasSize(3);
+    }
+
+    @DisplayName("요구사항2 - Set의 contains() 메소드를 활용해 1, 2, 3의 값이 존재하는지를 확인하는 학습테스트")
+    @ParameterizedTest(name = "{displayName} -> number: {0}")
+    @ValueSource(ints = {1, 2, 3})
+    void contains(int number) {
+        // given & when & then
+        assertThat(numbers.contains(number)).isTrue();
```

Comment:
```suggestion
        // when & then
        assertThat(numbers.contains(number)).isTrue();
```

소소하지만... ParameterizedTest로 테스트 데이터를 준비하기 때문에 해당 부분은 when & then 만 해당하지 않을까요!?

Reply:
테스트 데이터가 ParameterizedTest로 이미 준비된 상태로 생각하니 그렇네요!
다음 단계부터는 말씀해주신 내용으로 작성하겠습니다 ㅎㅎ
감사합니다 👍